tracing-core/src/field.rs

```diff
@@ -270,6 +273,12 @@ pub struct DisplayValue<T: fmt::Display>(T);
 #[derive(Clone)]
 pub struct DebugValue<T: fmt::Debug>(T);
 
+/// A 'Value' which serializes error and it's sources.
```

Comment:
```suggestion
/// A `Value` which serializes error and its sources.
```

Reply:
Also I'm not sure we should talk about sources here. What will be recorded is subscriber-specific, it can completely ignore the sources.